tests/actions/test_validation_operators_in_data_context.py

```diff
@@ -55,12 +55,11 @@ def validation_operators_data_context(
     df.expect_column_values_to_not_be_null(column="y")
     warning_expectations = df.get_expectation_suite(discard_failed_expectations=False)
 
-    data_context.save_expectation_suite(
-        failure_expectations, expectation_suite_name="f1.failure"
-    )
-    data_context.save_expectation_suite(
-        warning_expectations, expectation_suite_name="f1.warning"
-    )
+    failure_expectations.expectation_suite_name = "f1.failure"
+    data_context.add_expectation_suite(expectation_suite=failure_expectations)
+
+    warning_expectations.expectation_suite_name = "f1.warning"
+    data_context.save_expectation_suite(expectation_suite=warning_expectations)
```

Comment:
What is the difference between `save_expectation_suite` and `update_expectation_suite`?

Reply:
whoops this should also be `add` - same answer as above though